index.js

```diff
@@ -159,10 +182,13 @@ function buildRouteRate (pluginComponent, params, routeOptions) {
           params.onExceeded(req)
         }
 
-        if (params.addHeaders['x-ratelimit-limit']) { res.header('x-ratelimit-limit', maximum) }
-        if (params.addHeaders['x-ratelimit-remaining']) { res.header('x-ratelimit-remaining', 0) }
-        if (params.addHeaders['x-ratelimit-reset']) { res.header('x-ratelimit-reset', Math.floor(ttl / 1000)) }
-        if (params.addHeaders['retry-after']) { res.header('retry-after', params.timeWindow) }
+        if (params.addHeaders[params.labels.rateLimit]) { res.header(params.labels.rateLimit, maximum) }
+        if (params.addHeaders[params.labels.rateRemaining]) { res.header(params.labels.rateRemaining, 0) }
+        if (params.addHeaders[params.labels.rateReset]) { res.header(params.labels.rateReset, timeLeft) }
+        if (params.addHeaders[params.labels.retryAfter]) {
+          const resetAfterTime = (typeof params.enableDraftSpec === 'boolean' && params.enableDraftSpec) ? timeLeft : params.timeWindow
```

Comment:
do we need to check again if it's a boolean here?

Reply:
You are right, I removed it